cmd/evm/runner.go

```diff
@@ -70,14 +70,13 @@ func readGenesis(genesisPath string) *core.Genesis {
 	return genesis
 }
 
-func timedExec(bench bool, execFunc func() ([]byte, uint64, error)) ([]byte, uint64, time.Duration, error) {
-	var (
-		output   []byte
-		gasLeft  uint64
-		execTime time.Duration
-		err      error
-	)
+type ExecStats struct {
```

Comment:
Any reason for making this public?

Reply:
No, except I can name variables of this type as `execStats`.

Reply:
Do you want me to change that?

Reply:
> Any reason for making this public?

That's @karalabe s' way of saying "Please make this private, unles there's some really good reason otherwise"